.github/workflows/python_wheels.yml

```diff
@@ -0,0 +1,195 @@
+name: Build Python Wheels
+
+# TODO: Shrink this down to pushes to master, perhaps. During development, this
+# needs to run more often to work out bugs.
+on:
+  - push
```

Comment:
@Siltala, let me know what you would like to see here. Building and testing the wheels is pretty slow, since aarch64 architectures need to be emulated. The whole thing takes about 2-3 hours.

The current workflow doesn't actually upload the built wheels anywhere. https://cibuildwheel.readthedocs.io/en/stable/deliver-to-pypi has some guides on how to do that. I think it might be something like this:
```yaml
  build_sdist:
    # Make a plain source distribution with nothing precompiled.
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - name: Build SDist
      run: pipx run build --sdist
    - uses: actions/upload-artifact@v3
      with:
        name: sdist
        path: dist/*.tar.gz

  upload_all:
    # Upload all wheels and sdist to pypi
    needs: 
      - build_sdist
      - test_x86_wheels
      - test_linux_aarch64_wheels
    runs-on: ubuntu-latest
    if: github.event_name == 'release' && github.event.action == 'published'
    steps:
    - uses: actions/download-artifact@v3
      with:
        name: sdist
        path: ./sdist/
    - uses: actions/download-artifact@v3
      with:
        name: x86_wheels
        path: ./x86_wheels/
    - uses: actions/download-artifact@v3
      with:
        name: aarch64_wheels
        path: ./aarch64_wheels/
    - name: Combine artifacts
      run: |
        mkdir dist
        mv sdist/* dist
        mv x86_wheels/* dist
        mv aarch64_wheels/* dist
    - uses: pypa/gh-action-pypi-publish@v1.5.0
      with:
        user: __token__
        password: ${{ secrets.pypi_password }}
```

But of course I haven't tested that, because it needs a `pypi_password` to be set for the repository.

Reply:
I'll be happy to help with the PyPI side of this. It ought to be simple, I hope; the hard part is building the wheels.